_episodes/06-raster-intro.md

```diff
@@ -0,0 +1,462 @@
+---
+title: "Read and visualize raster data"
+teaching: 40
+exercises: 20
+questions:
+-  "What is a raster dataset?"
+-  "How do I read and plot raster data in Python?"
+-  "How can I handle missing values for a raster?"
+
+objectives:
+-  "Describe the fundamental attributes of a raster dataset."
+-  "Explore raster attributes and metadata using Python."
+-  "Read rasters into Python using the `rioxarray` package."
+-  "Visualize single-/multi- band raster data."
+
+keypoints:
+- "One can use `rioxarray.open_rasterio()` to read raster data"
+- "`rioxarray` stores CRS information as a CRS object that can be converted to an EPSG code or PROJ4 string."
+- "It is good to consider using nan to represent no data value(s), for statistics and visualization. "
+- "The `DataArray.plot()` function can be used to visualize single-band raster. Consider using the `robust=True` option for a better color limit."
+- "The `DataArray.plot.imshow()` function can be used to visualize multi-band raster."
+---
+
+> ## Things You'll Need To Complete This Episode
+>
+> See the [lesson homepage]({{ site.baseurl }}) for detailed information about the software,
+> data, and other prerequisites you will need to work through the examples in this episode.
+{: .prereq}
+
+In this episode, we will introduce the fundamental principles, packages and
+metadata/raster attributes that are needed to work with raster data in Python. We will discuss some of the core metadata elements that we need to understand to work with rasters, including Coordinate Reference Systems, no data values, and resolution. We will also explore missing and bad data values as stored in a raster and how Python handles these elements.
```

Comment:
```suggestion
metadata/raster attributes that are needed to work with raster data in Python. We will discuss some of the core metadata elements like Coordinate Reference Systems (CRS), no data values, and resolution. We will also explore missing and bad data values as stored in a raster and how Python handles these elements.
```

Reply:
here I suggested removing `that we need to understand to work with rasters, including` to avoid duplications. This part is already mentioned in the previous sentence.